bindings/pyroot/src/Pythonize.cxx

```diff
@@ -2515,6 +2529,9 @@ Bool_t PyROOT::Pythonize( PyObject* pyclass, const std::string& name )
          Utility::AddToClass( pyclass, "__setitem__", (PyCFunction) VectorBoolSetItem );
       }
 
+      // add pretty printing
+      Utility::AddToClass( pyclass, "__str__", (PyCFunction) ClingPrintValue);
```

Comment:
Can't this be done for all classes? why do we still need the "specializations" below for map, pair etc?

Reply:
Could be easily done, it's just not yet sure whether we really want to do this. We'll discuss this on Thursday in the PPP!